src/graphql/types/payload/user-quiz-questions-update-completed.ts

```diff
@@ -15,4 +15,4 @@ const UserQuizQuestionsUpdateCompletedPayload = new GT.Object({
   }),
 })
 
-export default UserQuizQuestionsUpdateCompletedPayload
+export default UserQuizQuestionUpdateCompletedPayload
```

Comment:
Rename this file too, and line 13: `type: UserQuizQuestion`

Reply:
done